tutorial/20_recipes/009_ask_and_tell.py

```diff
@@ -20,13 +20,16 @@
 Let's consider the traditional supervised classification problem; you aim to maximize the validation accuracy.
 To do so, you train `LogisticRegression` as a simple model.
 """
+import warnings
 import numpy as np
 from sklearn.datasets import make_classification
+from sklearn.exceptions import ConvergenceWarning
 from sklearn.linear_model import LogisticRegression
 from sklearn.model_selection import train_test_split
 
 import optuna
 
+warnings.filterwarnings("ignore", category=ConvergenceWarning, module="sklearn")
```

Comment:
The [configuration guide of Sphinx-gallery](https://sphinx-gallery.github.io/stable/configuration.html?highlight=warning#removing-warnings) suggests that warnings can be removed by adding similar change in `conf.py`.

It sounds reasonable since 1) adding warnings.filterwarnings makes code example long, and 2) we may filter similar warnings in the other examples if exist. What do you think of it?

Reply:
Lovely. I've updated PR changes by following your comment.